src/main/kotlin/com/openlattice/chronicle/storage/ChroniclePostgresTables.kt

```diff
@@ -180,6 +188,52 @@ class ChroniclePostgresTables {
             .primaryKey(USER_ID)
             .overwriteOnConflict()
 
+        @JvmField
+        val APPS_USAGE = PostgresTableDefinition("app_usage")
```

Comment:
I already implemented app usage and storage inside of redshift. Check out redshift tables class and make sure you aren't duplicating that code.

Reply:
Is this for storing the results of an app usage survey? If so I would update table name to reflect that so it is less confusing.

Reply:
Yes, storing results of survey